options/options.js

```diff
@@ -16,22 +23,27 @@ function save_options() {
 
 // Restores option state using the preferences stored in chrome.storage.
 function restore_options() {
-    chrome.storage.local.get('MB', function(items) {
-        var MB;
-        if(items.MB !== undefined) {
-            MB = items.MB;
+    chrome.storage.local.get(['DecoderMemoryLimitInMegabytes', 'DebugOutput'], function(items) {
```

Comment:
I would like it if we could consistently use `function( options ){ .. }` for these instead. Above I think you have `function( obj ){ .. }` for example and here it is `items` ...